how-to/how-to-use-virtio-mem-with-kata.md

```diff
@@ -0,0 +1,49 @@
+# Kata Containers with `virtio-mem`
+
+- [Introduction](#introduction)
+- [Requisites](#requisites)
+- [Run a Kata Container with utilizing `virtio-mem`](#run-a-kata-container-utilizing-virtio-mem)
+
+## Introduction
+
+The basic idea of `virtio-mem` is to provide a flexible, cross-architecture memory hot plug and hot unplug solution that avoids many limitations imposed by existing technologies, architectures, and interfaces. More details can be found in https://lkml.org/lkml/2019/12/12/681.
+
+Kata Containers with `virtio-mem` support memory resize.
+
+## Requisites
+
+Kata Containers with `virtio-mem` requires Linux and the QEMU that support `virtio-mem`.
+
+The Linux kernel is at https://github.com/davidhildenbrand/linux/tree/virtio-mem-rfc-v4.
+The Linux kernel config that can work with Kata Containers is at https://gist.github.com/teawater/016194ee84748c768745a163d08b0fb9.
+
+The QEMU is at https://github.com/teawater/qemu/tree/kata-virtio-mem. (The original source is at https://github.com/davidhildenbrand/qemu/tree/virtio-mem.  Its base version of QEMU cannot work with Kata Containers.  So merge the commit of `virtio-mem` to upstream QEMU.)
+
+Set Linux and the QEMU that support `virtio-mem` with following line in the Kata Containers QEMU configuration `configuration-qemu.toml`.
+```
+path = "qemu-dir"
```

Comment:
Can you should which sections these settings apply to (`[hypervisor.qemu]`?) Ideally show those section names in the code blocks.

Same comment for the options below.

Reply:
done.